src/Workspaces/Core/Portable/CodeGeneration/CodeGenerationSymbolFactory.cs

```diff
@@ -195,7 +195,8 @@ public static bool IsCodeGenerationSymbol(this ISymbol symbol)
             CodeGenerationOperatorKind operatorKind,
             ImmutableArray<IParameterSymbol> parameters,
             ImmutableArray<SyntaxNode> statements = default,
-            ImmutableArray<AttributeData> returnTypeAttributes = default)
+            ImmutableArray<AttributeData> returnTypeAttributes = default,
+            string? documentationCommentXml = null)
```

Comment:
Why we need this string?

Reply:
effectively, we are synthesizing special members for symbols that the compiler does not actually ever create symbols for (unfortunately).  Things includes things like lifted operators or conversions.  Because our entire completion pipeline is built off of symbols, i'm implementing this functionality by synthesizing symbols on demand for these guys and passing them along.  Because we also want to show doc commetns for these synthesized symbols, this value gets passed along here.